### DISCUSSION TITLE
Issue with GHOSTS Documentation Link

### DISCUSSION THREAD

**Discussion by crispy6** [cmu-sei/GHOSTS#171] [General]
Hello! I tried to follow this link (https://cmu-sei.github.io/ghosts/) but received a 404 error. I am looking for further information on the specs for the VM that the GHOSTS command server would run on. I am hoping that the link provided is what will contain that information, and if so, that the 404 issue can be resolved.

**Comment by sei-dupdyke** [comment]
Hi - the link is https://cmu-sei.github.io/GHOSTS/ - sorry, github pages are case-sensitive.

Where did you see the lowercase version please?

**Reply by crispy6** [reply]
Thank you for the swift response! I found that link under Documentation at this webpage: https://cmu-sei.github.io/GHOSTS/
Which appears to just be a link back to the same webpage.

**Comment by sei-dupdyke** [comment]
Thanks, I found and fixed this with #[967bef1](https://github.com/cmu-sei/GHOSTS/commit/967bef1ddb5ae00738f9be2fab02126c7447cf4a). Sorry for the confusion.
